src/main/java/org/openpnp/machine/reference/ReferenceNozzleTipCalibration.java

```diff
@@ -603,7 +603,7 @@ public static void resetAllNozzleTips() {
         // reset all nozzle tip calibrations, as they have become invalid
         for (Head head : Configuration.get().getMachine().getHeads()) {
             for (Nozzle nozzle  : head.getNozzles()) {
-                for (NozzleTip nt : nozzle.getNozzleTips()) {
+                for (NozzleTip nt : nozzle.getCompatibleNozzleTips()) {
```

Comment:
And again, as above, changed from getNozzleTips to getCompatibleNozzleTips.

Reply:
It can go directly to the nozzle tips. See my push.